packages/edit-site/src/components/global-styles/preview-wrapper.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__unstableIframe as Iframe,
+	__unstableEditorStyles as EditorStyles,
+	privateApis as blockEditorPrivateApis,
+} from '@wordpress/block-editor';
+import { __unstableMotion as motion } from '@wordpress/components';
+import {
+	useThrottle,
+	useReducedMotion,
+	useResizeObserver,
+} from '@wordpress/compose';
+import { useLayoutEffect, useState, useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { useGlobalStyle, useGlobalStylesOutput } = unlock(
+	blockEditorPrivateApis
+);
+
+const firstFrameVariants = {
```

Comment:
If the props were changed from `firstFrame, etc..` to an array, do you think it's reasonable for these variants to be passed as part of the frame components?

Might make for a neater `frames.map` in the render block below (inside `<motion.div initial="start" ... />`)?

Then it would be able to accommodate one or more frames. 🤷🏻